src/chrome/content/rules/Bilibili.com.xml

```diff
@@ -13,16 +11,16 @@
 		- space
 		- yoo
 		
-	Passive mixed content ON
-		- app, pay, passport FROM static.hdslb.com
-		- bangumi            FROM i[0-2].hdslb.com
-	Display mixed content ON
-		- app           FROM activity.hdslb.com
-		- bangumi       FROM static.hdslb.com, s[1-3].hdslb.com
-		- pay, passport FROM api*
-	* Secured by us
```

Comment:
`hdslb.com` supports https now, I think we can create a mixcontent ruleset for them.